verticapy/tests_new/machine_learning/vertica/__init__.py

```diff
@@ -17,48 +17,497 @@
+        "rmse": {"rel": REL_TOLERANCE, "abs": ABS_TOLERANCE},
+        **dict.fromkeys(["r2", "R-squared"], {"rel": 1e-05, "abs": ABS_TOLERANCE}),
+        **dict.fromkeys(
+            ["r2_adj", "Adj. R-squared"], {"rel": 1e-05, "abs": ABS_TOLERANCE}
+        ),
+        "aic": {"rel": 1e-05, "abs": ABS_TOLERANCE},
+        "bic": {"rel": REL_TOLERANCE, "abs": ABS_TOLERANCE},
+        "F-statistic": {"rel": 1e-04, "abs": ABS_TOLERANCE},
+        "Prob (F-statistic)": {"rel": REL_TOLERANCE, "abs": ABS_TOLERANCE},
+        "Kurtosis": {"rel": 1e-02, "abs": ABS_TOLERANCE},
+        "Skewness": {"rel": 1e-03, "abs": ABS_TOLERANCE},
+        "Jarque-Bera (JB)": {"rel": 1e-02, "abs": ABS_TOLERANCE},
+        "df": {"rel": REL_TOLERANCE, "abs": ABS_TOLERANCE},
+        "ss": {"rel": 1e-04, "abs": ABS_TOLERANCE},
+        "ms": {"rel": 1e-03, "abs": ABS_TOLERANCE},
+        "f": {"rel": 1e-04, "abs": ABS_TOLERANCE},
+        "p_value": {"rel": REL_TOLERANCE, "abs": ABS_TOLERANCE},
+        "coef_": {"rel": 1e-03, "abs": ABS_TOLERANCE},
+        "intercept_": {"rel": 1e-04, "abs": ABS_TOLERANCE},
+        "score": {"rel": 1e-01, "abs": ABS_TOLERANCE},
+        "predict": {"rel": REL_TOLERANCE, "abs": ABS_TOLERANCE},
+        "to_python": {"rel": REL_TOLERANCE, "abs": ABS_TOLERANCE},
+        "load_model": {"rel": REL_TOLERANCE, "abs": ABS_TOLERANCE},
+    },
+    "AR": {
+        "explained_variance": {"rel": 1e-04, "abs": ABS_TOLERANCE},
+        "max_error": {"rel": 1e-02, "abs": ABS_TOLERANCE},
+        "median_absolute_error": {"rel": 1e-01, "abs": ABS_TOLERANCE},
+        "mean_absolute_error": {"rel": 1e-02, "abs": ABS_TOLERANCE},
+        "mean_squared_error": {"rel": 1e-02, "abs": ABS_TOLERANCE},
+        "mean_squared_log_error": {"rel": 1e-01, "abs": ABS_TOLERANCE},
+        "rmse": {"rel": 1e-02, "abs": ABS_TOLERANCE},
+        **dict.fromkeys(["r2", "R-squared"], {"rel": 1e-03, "abs": ABS_TOLERANCE}),
+        **dict.fromkeys(
+            ["r2_adj", "Adj. R-squared"], {"rel": 1e-03, "abs": ABS_TOLERANCE}
+        ),
+        "aic": {"rel": 1e-02, "abs": ABS_TOLERANCE},
+        "bic": {"rel": 1e-01, "abs": ABS_TOLERANCE},
+        "phi_": {"rel": 1e-01, "abs": ABS_TOLERANCE},
+        "intercept_": {"rel": REL_TOLERANCE, "abs": ABS_TOLERANCE},
+        "mse_": {"rel": 1e-02, "abs": ABS_TOLERANCE},
+        "predict": {"rel": 1e-01, "abs": ABS_TOLERANCE},
+    },
+    "MA": {
+        "explained_variance": {"rel": 1e-01, "abs": ABS_TOLERANCE},
+        "max_error": {"rel": 1e-00, "abs": ABS_TOLERANCE},
+        "median_absolute_error": {"rel": 1e-01, "abs": ABS_TOLERANCE},
+        "mean_absolute_error": {"rel": 1e-01, "abs": ABS_TOLERANCE},
+        "mean_squared_error": {"rel": 1e-01, "abs": ABS_TOLERANCE},
+        "mean_squared_log_error": {"rel": 1e-01, "abs": ABS_TOLERANCE},
+        "rmse": {"rel": 1e-01, "abs": ABS_TOLERANCE},
+        **dict.fromkeys(["r2", "R-squared"], {"rel": 1e-01, "abs": ABS_TOLERANCE}),
```

Comment:
Why does ARMA have a much looser tolerance than AR or ARMA? Did you try tighter?

Reply:
same as first comment.